src/Avalonia.Base/Threading/DispatcherOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Avalonia.Threading;
+
+/// <summary>
+/// AppBuilder options for configuring the <see cref="Dispatcher"/>.
+/// </summary>
+public class DispatcherOptions
+{
+    /// <summary>
+    /// Gets or sets a timeout after which the dispatcher will start prioritizing input events over
+    /// rendering. The default value is 1 second.
+    /// </summary>
+    /// <remarks>
+    /// If no input events are processed within this time, the dispatcher will start prioritizing
+    /// input events over rendering to prevent the application from becoming unresponsive. This may
+    /// need to be lowered on resource-constrained platforms where input events are processed on
+    /// the same thread as rendering.
+    /// </remarks>
+    public TimeSpan InputStarvationTimeout { get; set; } = TimeSpan.FromSeconds(1);
```

Comment:
```suggestion
    public TimeSpan InputStarvationTimeout { get; init; } = TimeSpan.FromSeconds(1);
```

Reply:
None of our other `*Options` classes uses `init` (because they predate `init` properties). Not sure if it's better to make new options properties init-only and leave the others, or convert them all for 12.0. Not very important IMO though in the sense that `init` won't solve any real-world problem here.

Reply:
As long as these options are only read once by the backend they can be mutable.